.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
               "sacc=0.2"
 
             # requires special compilers we don't have
-            # pip install cosmosis-standalone==0.0.9
+            pip install cosmosis-standalone==0.2.2
```

Comment:
You need to do this in conda.

Reply:
So add to the list above

Reply:
why?  I've never understood when to pip install and when to conda install.

Reply:
pip will pull other deps from pip when they could be in conda, which can end up with messed up envs later. You always use conda first, and then pip, preferably with --no-deps, second. This is a generic recommendation for conda.